src/main/scala/com/danielasfregola/zucchini/Dictionary.scala

```diff
@@ -16,6 +18,10 @@ object Dictionary extends App {
     texts.flatMap(line => line.split("\\W+"))
   }
 
-  println(extractUniqueWordsFromTextFile("BaconipSum.txt").mkString(","))
+  //  println(extractUniqueWordsFromTextFile("BaconipSum.txt").mkString(","))
 
+  val words: Set[String] = extractUniqueWordsFromTextFile("BaconipSum.txt")
```

Comment:
Although this is correct, you should change this so that the file name is given rather than fixed.

Maybe, you could create a class that given a text file, gives you an `Arbitrary[String]` based on the words in the given file. Your class should also handle corner cases. For example, what would happen if the given file does not exist?

Cheers,
Daniela